benchmarking/smartseq2/merge_expression_matrix/merge_table_quants.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import pandas as pd
+import numpy as np
+from google.cloud import storage
+import json
+from os.path import basename
+import sys
+import requests
+
+
+def merge_expression_matrix(uuid, run_name,output_name,value_name):
```

Comment:
Thanks for the comments, Jishu. I'm better understanding what you're doing here now. Armed with that knowledge, I think it would be nice to have more documentation here about what the input and output look like, since there is no inspectable return value. 

My preference would be to break up this function into a few sub-functions, as it would make some of the work more reusable, and create some intermediate landmarks where the user can see what the results are. 

I might break this up as: 
`parse_sample_uuid()`
`merge_expression_vectors()`
`count_to_tpm()` or `parse_feature_counts_output()`
`parse_rsem_output()`
`write_output()`

I wonder if the `feature_counts` and `rsem` values should be in separate functions? Do you ever expect to merge outputs from both of them? It sounds like the answer is no, from your docstring. Alternatively, perhaps we should remove some of the conditional testing and have an argument added to the command line parser that lets the user specify if they are merging rsem or featureCounts.

Reply:
yes. if we re-run pipelines testing (if we'll  have the third dataset soon), we will re-run everything, include STAR, HISAT2, RSEM FeatureCounts....So yes, we do need FeatureCounts as option.